### DISCUSSION TITLE
külső antenna csatlakozás

### DISCUSSION THREAD

**Discussion by PereszlenyiArpadECR-Trade** [nav-gov-hu/eRECEIPT#3] [Kérdések és válaszok]
Jelenleg a rendeletben az alábbi előírás van:
3. melléklet, FD) 14. Az AEE-t alkalmassá kell tenni külső antenna csatlakoztatására. Az antenna csatlakozót a pénztárgép külső burkolatára kell elhelyezni, és védett csavarkötéssel kell rögzíteni.
Ez hogyan változik az új rendeletben? Elhagyható a külső védett csavarkötés megoldás, lehet csak belő antenna, stb.?

**Comment by renced42** [comment]
Kedves @PereszlenyiArpadECR-Trade 
A jelenlegi tervezetben szerepel az előírás, megvizsgáljuk a kérdést.
